src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -159,6 +159,35 @@ def test_get_unique_name(self, cephadm_module):
         new_mgr = cephadm_module.get_unique_name('mgr', 'myhost', existing)
         match_glob(new_mgr, 'myhost.*')
 
+    @mock.patch("cephadm.serve.CephadmServe._run_cephadm", _run_cephadm('[]'))
+    def test_valid_url(self, cephadm_module):
+        # Test with valid URLs
```

Comment:
let's use `endpoint` here (both in function name and comment) instead of URL as technically these are not urls. 

BTW: what happen if the target is ipv6?

Reply:
Updated